ui/packages/shared/components/src/DateTimeRangePicker/DateTimeRangePickerPanel/AbsoluteDatePicker/index.tsx

```diff
@@ -17,8 +17,6 @@ import {AbsoluteDate, DateTimeRange, getDateHoursAgo} from '../../utils';
 import Button from '../../../Button';
 import {UTCDateTimePicker} from '../../../DateTimePicker';
 
-import 'react-datepicker/dist/react-datepicker.css';
```

Comment:
Removing this line here and placing it in the `ui/packages/app/web/src/App.tsx` file should hopefully solve the [css-npm](https://nextjs.org/docs/messages/css-npm) error seen on cloud.

Reply:
But this would be an inconvenience to make the consuming project to import this css file explicitly. 

It's ok for now, in the future we can look into an alternative way to import those styles into `DatePicker` component only so that the consumers don't need to worry about it.

Reply:
Yeah it was the most ok solution I could think of for now. Definitely will revisit in the future.